### PR TITLE
12.0 merge from 11.0

### PR DESCRIPTION
**Pull request by em230418**

### CODE REVIEW

File: website_multi_company/doc/changelog.rst [hunk 1]
```diff
@@ -1,4 +1,9 @@
-`2.4.0`
+`3.0.1`
+-------
+
+- **Fix:** Non-admin users now have access to read company logo in a website
+
+`3.0.0`
```
Comment (kaadevelop) [inline website_multi_company/doc/changelog.rst]: Has to be tested

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.com/it-projects-llc/website-addons.svg?branch=10.0)](https://travis-ci.com/it-projects-llc/website-addons)
```
Comment (yelizariev) [inline README.md]: Please update change version to 12.0

File: website_multi_company/doc/changelog.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ramil-Mukhametzyanov) [reply website_multi_company/doc/changelog.rst]: :+1: checked

[the remark this answers is not in the document]
Comment (Ramil-Mukhametzyanov) [reply website_multi_company/doc/changelog.rst]: :+1: checked

**Comment by em230418** [comment]
I approve to merge it now

**Pull request merged**